src/main/java/com/seailz/discordjv/model/emoji/Emoji.java

```diff
@@ -111,9 +110,8 @@ public static Emoji decompile(JSONObject obj, DiscordJv discordJv) {
         return new Emoji(id, name, roles, user, requireColons, managed, animated, available);
     }
 
-    @Override
```

Comment:
@NotNull & add documentation

Reply:
`@NotNull` is not here anymore

adding documentation